### PR TITLE
Update index.d.ts for tween.js version 17.2.0

### PR DESCRIPTION
**Pull request by vldk**
Added definitions for new methods at Tween: 
1.   getId(): number;
2.   isPlaying():boolean;
3.   group(group: Group): Tween;

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
1. https://github.com/tweenjs/tween.js/commit/765d0487e299db2c83600825b4afb97e81ebac8f
2. https://github.com/tweenjs/tween.js/commit/6df6ae1de3677468bd59033ebf19813ef83b12bf
3. https://github.com/tweenjs/tween.js/commit/2f1b1b624770696e8a64c24d0a5052b84a841fa6

- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

**Review by alexburner — APPROVED** [review]
This LGTM

Technically, all of the method signatures should be returning `this` instead of `Tween`, but that's an existing issue and this PR maintains consistency with the existing types.

File: types/tween.js/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for tween.js v16.9.0
+// Type definitions for tween.js v17.2.0
```
Comment (uniqueiniquity) [inline types/tween.js/index.d.ts]: This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `// Type definitions for tween.js 17.2`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/tween.js/index.d.ts (hunk 1 above)
Comment (vldk) [reply types/tween.js/index.d.ts]: Done

**Review by uniqueiniquity — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**